Scripts/Swift/SetUpDeliveryConstants.swift

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/swift
+
+import Foundation
+
+let fileManager = FileManager.default
+
+print("Do you want to set up Constants values? (Can be edited later) [Y/n]: ")
+
+var arg = "y"
+
+arg = readLine() ?? arg
+
+switch arg.lowercased() {
+    case "y", "yes", "":
+        let error = try safeShell("open -a NotXcode fastlane/Constants/Constant.swift")
```

Comment:
Please put it back to `Xcode` after testing 😅

Reply:
😆